backend/api/api.py

```diff
@@ -77,7 +77,17 @@ async def add_result(
     test_name: str, data: TestResults, user: User = Depends(auth.current_active_user)
 ):
     store = DBStore()
-    await store.add_results(user, test_name, data.root)
+    try:
+        await store.add_results(user, test_name, data.root)
+    except DBStoreResultExists:
+        raise HTTPException(status_code=400, detail="Result already exists")
```

Comment:
Is it feasible to print like the primary key fields of the duplicate that fails here? If user uploads a batch of results, it's annoying not to know which one to fix.